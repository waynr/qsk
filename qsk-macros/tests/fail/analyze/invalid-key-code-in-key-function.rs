use qsk_macros::remap;

fn main() {
    remap!(
        ModLayer: {
            F -> TT(Navigation, MEOW),
        },
    );
}
