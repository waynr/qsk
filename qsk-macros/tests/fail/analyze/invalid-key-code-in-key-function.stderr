error: invalid key code when converting parse::KeyFunctionParameter to KeyCode: VariantNotFound
 --> tests/fail/analyze/invalid-key-code-in-key-function.rs:6:33
  |
6 |             F -> TT(Navigation, MEOW),
  |                                 ^^^^
